pdbr/_pdbr.py

```diff
@@ -407,4 +429,28 @@ def print_stack_entry(self, frame_lineno, prompt_prefix="\n-> ", context=None):
             self.shell.hooks.synchronize_with_editor(filename, lineno, 0)
             # vds: <<
 
+        def run_magic(self, line) -> str:
+            """
+            Parses the line and runs the appropriate magic function.
+            Assumes that the line is without a leading '%'.
+            """
+            magic_name, arg, line = self.parseline(line)
+            if hasattr(self, f"do_{magic_name}"):
+                # We want to use do_{magic_name} methods if defined.
+                # This is indeed the case with do_pdef, do_pdoc etc,
+                # which are defined by our base class (IPython.core.debugger.Pdb).
+                result = getattr(self, f"do_{magic_name}")(line)
+                return result or ""
```

Comment:
```suggestion
                return self._print(result) or ""
```
this change will fix some commands like `lsmagic`